database/src/main/resources/db/migration/V1.95.0__update_drawer_notification_insert_function.sql

```diff
@@ -0,0 +1,18 @@
+CREATE OR REPLACE PROCEDURE insert_drawer_notifications_with_id(userArray text, pOrgId text, pEventId uuid, pCreated timestamp)
+    LANGUAGE plpgsql
+AS $$
+DECLARE
+    userName text;
+BEGIN
+    FOREACH userName IN ARRAY (string_to_array(userArray,','))
+        LOOP
+            INSERT INTO drawer_notification(user_id, org_id, event_id, created)
+            VALUES (userName, pOrgId, pEventId, pCreated)
+            ON CONFLICT (org_id, user_id, event_id) DO NOTHING;
+        END LOOP;
```

Comment:
This executes `n` INSERT queries with `n` the number of users.

Is there a way to execute one INSERT query with `n` VALUES? That would probably be much faster.

Reply:
According local tests, it's almost 30% faster with the new syntax :)